gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -70,6 +73,22 @@
     this.channel = ghfs.getGcsFs().open(gcsPath, readOptions);
   }
 
+  GoogleHadoopFSInputStream(
+          GoogleHadoopFileSystemBase ghfs,
+          URI gcsPath,
+          GoogleCloudStorageItemInfo itemInfo,
+          GoogleCloudStorageReadOptions readOptions,
+          FileSystem.Statistics statistics)
+          throws IOException {
+    logger.atFiner().log(
+            "GoogleHadoopFSInputStream(gcsPath: %s, readOptions: %s)", gcsPath, readOptions);
```

Comment:
might be useful to log `itemInfo` as well for debugging.

Do we need a null check on `itemInfo` here ?

nit: re-use the constructor at line 62 to set the rest of the fields